test/powershell/Modules/Microsoft.PowerShell.Security/AmsiInterface.Tests.ps1

```diff
@@ -0,0 +1,46 @@
+﻿##
+## AMSI Interface Test
+##
+## Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
For consistency (in case we ever need to update the header), can you use the exact form of this header and ensure it's the top two lines first.

Reply:
Done